src/main/java/de/dhbw/karlsruhe/students/mailflow/core/application/email/provide/UCCollectionProvideEmails.java

```diff
@@ -0,0 +1,26 @@
+package de.dhbw.karlsruhe.students.mailflow.core.application.email.provide;
+
+import de.dhbw.karlsruhe.students.mailflow.external.infrastructure.email.parsing.FileMailboxRepository;
+
+/**
+ * As we got many useCases to provide emails, we need a collection to reduce the amount of
+ * parameters
+ *
+ * @author Jonas-Karl
+ */
+public record UCCollectionProvideEmails(
```

Comment:
do we still use records or do we make the class final ?
I know a record is the best practise for immutable objects, but i would say that a final class with get, hashcode etc methods will be more code

Reply:
Currently we are still using records. I suggest that we change all records at once in the last few days of this project. For now it helps us in the development process by implicitly creating the getter-methods